configureWorkspace/configure.ts

```diff
@@ -415,8 +412,11 @@ async function configureCore(actionContext: IActionContext, options: ConfigureAp
 
         if (writeFile) {
             // Paths in the docker files should be relative to the Dockerfile (which is in the output folder)
-            fs.writeFileSync(filePath, generatorFunction(serviceNameAndPathRelativeToOutput, platformType, os, port, packageInfo), { encoding: 'utf8' });
-            filesWritten.push(fileName);
+            let fileContents = generatorFunction(serviceNameAndPathRelativeToOutput, platformType, os, port, packageInfo);
+            if (fileContents) {
+                fs.writeFileSync(filePath, fileContents, { encoding: 'utf8' });
+                filesWritten.push(fileName);
+            }
```

Comment:
This looks like a bug that would have affected other usecases too. Nice catch.

Reply:
You mean if (fileContents)?
This is actually just a change in behavior - moving special-case code out of the common code.